src/NuGet.Clients/NuGet.Console/WpfConsole/WpfConsoleKeyProcessor.cs

```diff
@@ -346,7 +346,7 @@ protected override int InternalExec(ref Guid pguidCmdGroup, uint nCmdID, uint nC
                                 ExecuteCommand(VSConstants.VSStd2KCmdID.END);
                                 ExecuteCommand(VSConstants.VSStd2KCmdID.RETURN);
 
-                                WpfConsole.EndInputLine();
+                                NuGetUIThreadHelper.JoinableTaskFactory.RunAsync(() => EndInputLineAsync(WpfConsole));
```

Comment:
PostOfFault.   Why didn't the threading analyzer warn you of this? 😞   Maybe my "prefer VS SDK" PR will fix that.

Reply:
The dev base that I was working on top of was not warning for whatever reason. 
If you notice, my build was green. 

After rebasing on top of latest dev, I got the warning and I fixed it.